kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -329,6 +331,12 @@ def kube_pod_status_phase(self, message, **kwargs):
                                          tags=[self._label_to_tag("pod", metric.label),
                                                self._label_to_tag("namespace", metric.label)] + self.custom_tags)
 
+            tags = []
+            for label in metric.label:
+                if label.name in ["pod", "namespace", "phase"]:
+                    tags += [self._format_tag(label.name, label.value)]
```

Comment:
Doesn't `_label_to_tag` do the same thing as the for loop? Can we do something like this?

```python
tags = [self._label_to_tag("pod", metric.label), self._label_to_tag("namespace", metric.label), 
        self._label_to_tag("phase", metric.label)] + self.custom_tags
```
And then we can just do `tags=tags` for both `_condition_to_tag_check` and `gauge` calls.

Reply:
You're absolutely right, that's a much nicer way to do things. Updating now.